vision/object_detection_segmentation/retinanet/dependencies/retinanet-export.py

```diff
@@ -0,0 +1,187 @@
+
+    model_dir = os.path.join(dir, 'model.onnx')
+    torch.onnx.export(model, inputs, model_dir, verbose=True, input_names=input_names,
+                      output_names=output_names, example_outputs=outputs, **kwargs)
+
+    test_data_dir = os.path.join(dir, data_dir)
+    if not os.path.exists(test_data_dir):
+        os.makedirs(test_data_dir)
+
+    save_data(test_data_dir, "input", input_names, inputs_flatten)
+    save_data(test_data_dir, "output", output_names, outputs_flatten)
+
+    return model_dir, test_data_dir
+
+
+def inference(file, inputs, outputs):
+    inputs_flatten = flatten(inputs)
+    inputs_flatten = update_flatten_list(inputs_flatten, [])
+    outputs_flatten = flatten(outputs)
+    outputs_flatten = update_flatten_list(outputs_flatten, [])
+
+    sess = onnxruntime.InferenceSession(file)
+    ort_inputs = dict((sess.get_inputs()[i].name, to_numpy(input)) for i, input in enumerate(inputs_flatten))
+    res = sess.run(None, ort_inputs)
+
+    if outputs is not None:
+        print("== Checking model output ==")
+        [np.testing.assert_allclose(to_numpy(output), res[i], rtol=1e-03, atol=1e-05) for i, output in enumerate(outputs_flatten)]
+        print("== Done ==")
+
+
+def perf_run(sess, feeds, min_counts=5, min_duration_seconds=10):
+    # warm up
+    sess.run([], feeds)
+
+    start = timer()
+    run = True
+    count = 0
+    per_iter_cost = []
+    while run:
+        iter_start = timer()
+        sess.run([], feeds)
+        end = timer()
+        count = count + 1
+        per_iter_cost.append(end - iter_start)
+        if end - start >= min_duration_seconds and count >= min_counts:
+            run = False
+    return count, (end - start), per_iter_cost
+
+
+def torch_inference(model, input):
+    print("====== Torch Inference ======")
+    output=model(input)
+    return output
+
+
+def ort_inference(file, inputs_flatten, outputs_flatten):
+    print("====== ORT Inference ======")
+    ort_sess = onnxruntime.InferenceSession(file)
+    ort_inputs = dict((ort_sess.get_inputs()[i].name, to_numpy(input)) for i, input in enumerate(inputs_flatten))
+    ort_outs = ort_sess.run(None, ort_inputs)
+    if outputs_flatten is not None:
+        print("== Checking model output ==")
+        [np.testing.assert_allclose(to_numpy(output), ort_outs[i], rtol=1e-03, atol=1e-05) for i, output in
+         enumerate(outputs_flatten)]
+    print("== Done ==")
+
+
+def get_image_from_url(url, size=None):
+    import requests
+    from PIL import Image
+    from io import BytesIO
+    from torchvision import transforms
+
+    data = requests.get(url)
+    image = Image.open(BytesIO(data.content)).convert("RGB")
+    image = image.resize(size, Image.BILINEAR)
+
+    to_tensor = transforms.ToTensor()
+    return to_tensor(image)
+
+
+def get_test_images():
+    image_url = "http://farm3.staticflickr.com/2469/3915380994_2e611b1779_z.jpg"
```

Comment:
does it make sense to save this image locally in the dependencies folder of the repo?